cmake/modules/Hexagon.cmake

```diff
@@ -34,7 +34,7 @@ function(find_hexagon_toolchain)
     set(TRY_PATH "${USE_HEXAGON_SDK}")
   endif()
   message(STATUS "Looking for Hexagon toolchain in ${TRY_PATH}")
-  tvm_file_glob(GLOB_RECURSE HEXAGON_CLANG "${TRY_PATH}/*/hexagon-clang++")
+  file(GLOB_RECURSE HEXAGON_CLANG "${TRY_PATH}/*/hexagon-clang++")
```

Comment:
Just curious: why replace `tvm_file_glob` with `file`?

Reply:
`tvm_file_glob` is supposed to be used on TVM sub-directoy files.